packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -43,7 +43,7 @@ export default decorate([
       editRemote: ({ reset }) => state => {
         const {
           remote,
-          domain = remote.domain,
+          domain = remote.domain || '',
```

Comment:
Can you explain this change?

Reply:
`domain` could theoretically be `undefined` (even though it shouldn't happen too much since it's HTML-required during the creation) so this avoids filling the input with an `"undefined"` string when editing the remote.